app/views/layouts/application.html.haml

```diff
@@ -13,4 +13,5 @@
     = render 'layouts/nav'
     = render 'layouts/flash'
     = yield
-    %script{:src => "https://code.jquery.com/jquery-2.1.1.min.js", :type => "text/javascript"}
+    %script{ :src => "https://code.jquery.com/jquery-2.1.1.min.js", :type => "text/javascript"}
+    %script{ :src => "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js"}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Line is too long. [90/80]